examples/workflows/fcn.py

```diff
@@ -41,13 +40,7 @@
 
 # Use IC method to get data source, this will regrid the data if needed
 time = datetime.datetime(2018, 1, 1)
-data_source = ic(
-    time,
-    sfno_inference_model.grid,
-    n_history=0,
-    channel_set=sfno_inference_model.channel_set,
-    source=schema.InitialConditionSource.cds,
-)
+data_source = cds.DataSource(channel_names=sfno_inference_model.in_channel_names)
```

Comment:
Thoughts on `from earth2mip.initial_conditions import CDSDatasource` instead of cds.DataSource?

Reply:
https://github.com/NVIDIA/earth2mip/pull/64#discussion_r1362482330